### PR TITLE
Support ECL 1.5

### PR DESCRIPTION
**Pull request by cmark**
This PR migrates the current grammar to support the latest 1.5 ECL specification:
https://confluence.ihtsdotools.org/download/attachments/33493263/doc_ExpressionConstraintLanguage_v1.5-en-US_INT_20210215.pdf?api=v2

### CODE REVIEW

**Comment by apeteri** [comment]
Grammar-wise it looks OK, but tests are complaining.

File: com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/Ecl.xtext [hunk 1]
```diff
@@ -122,87 +127,132 @@ Comparison:
...
+	| LanguageFilter 
+	| TypeFilter 
+	| DialectFilter 
+	// QL 0.1 filters (ECL extensions)	 
+	| ActiveFilter 
+	| PreferredInFilter
+	| AcceptableInFilter
+	| LanguageRefSetFilter
+	| ModuleFilter
+	| CaseSignificanceFilter
```
Comment (cmark) [inline com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/Ecl.xtext:184]: Could you please add the following filters as well? 🙇
* SemanticTagFilter (supported domains are `Concept` and `Description`)
* EffectiveTimeFilter (supported domains are `Concept` and `Description`)

**Review by cmark — COMMENTED** [review]
LGTM! Looks great! My only comment is to add two additional filters as extensions to the ECL 1.5 spec. Please find it inline.

**Review by cmark — COMMENTED** [review]
LGTM!

**Pull request merged**